agent/bin/test_install

```diff
@@ -8,6 +8,9 @@ mkdir -p build/fixtures
 source "$JAVA_HOME/release"
 export JAVA_VERSION
 
+source "test/helper.bash"
+export ANNOTATION_JAR="$(find_annotation_jar)"
```

Comment:
Seems like this commit is out of order with the other ones — shouldn't it come last?

Reply:
I don't think so. The tests in the `@NoAppMap` commit wouldn't work without these changes. What is it about this commit that makes you think it should come last?

Reply:
Ah, nevermind, for some reason I thought the changes there introduced the annotations jar, but I see it's been there before.